lib/route.js

```diff
@@ -196,13 +196,15 @@ function buildRouting (options) {
         opts.attachValidation = false
       }
 
+      if (!prefixing) {
```

Comment:
1. I'd rather see `prefixing === false`
2. If I'm not mistaken, `prefixing === false` is the default case. Can the code be refactored to move the default case outside of a conditional? I'd rather see the exceptional case in the conditional.

Reply:
Or just `(prefixing===true) return` to avoid the nesting

Reply:
I too would love to see the exceptional case in conditional block. However, early return doesn't seem to be an option because there's a lot of code after line 211 that seems necessary. 

I'm open to ideas for refactoring.

Reply:
>lot of code after line 211 that seems necessary.

Oh, it was my oversight

Reply:
Can we at least make the check explicit?

Reply:
Done.